src/Sinch/Verification/Common/VerificationMethod.cs

```diff
@@ -1,11 +1,13 @@
-﻿using Sinch.Core;
+﻿using System.Text.Json.Serialization;
+using Sinch.Core;
 
-namespace Sinch.Verification
+namespace Sinch.Verification.Common
 {
     /// <summary>
     ///     The method of the verification request.
     /// </summary>
     /// <param name="Value"></param>
+    [JsonConverter(typeof(EnumRecordJsonConverter<VerificationMethod>))]
```

Comment:
https://developers.sinch.com/docs/verification/api-reference/verification/tag/Verification-callbacks/ is defining 4th value for response: `seamless`